list.js

```diff
@@ -31,19 +31,19 @@
     };
 
     var doSearch = function(events) {
-        var $el = $('#calendarList');
```

Comment:
If wordpress is calling `jQuery.noConflict`, instead of having to change `$` to `jQuery`, we can pass in `jQuery` to the surrounding self-executing function, like this:

```js
(function ($) {
    ... list.js ...
})(jQuery);
```

http://stackoverflow.com/a/19491790/173630